backend/fixtures/1-site.json

```
@@ -9,7 +9,7 @@
     "gaw": null,
     "dvasId": "INO",
     "country": "Romania",
-    "iso_3166_1_alpha_2": "RO",
+    "countryCode": "RO",
     "iso_3166_2": null,
```

Uusi nimi tänne ja muualle:

```suggestion
    "countrySubdivisionCode": null,
```